### PR TITLE
Added option to explain how to load from a private git host

### PR DESCRIPTION
**Pull request by StevenCostiou**

### CODE REVIEW

File: General/Baselines.md [hunk 1]
```diff
@@ -771,20 +771,21 @@ The repository parameter is a string that can takes different form in case we ha
 
 The repository parameter to load a project from github/gitlab/bitbucket takes this form:
 
-`{prefix}://{owner}/{projectName}:{version}/{subFolder}`
+`{prefix}://{option}:{owner}/{projectName}:{version}/{subFolder}`
```
Comment (juliendelplanque) [inline General/Baselines.md]:
```suggestion
`{prefix}://{optionalHostname}:{owner}/{projectName}:{version}/{subFolder}`
```

File: General/Baselines.md [hunk 2]
```diff
@@ -771,20 +771,21 @@ The repository parameter is a string that can takes different form in case we ha
...
 
 This snippet should be configured with:
 
 - `{prefix}`: This is host-specific:
   - `github` for github
   - `bitbucket` for bitbucket
   - `gitlab` for gitlab
+- `{option}`: Optional server host, for private git servers
```
Comment (juliendelplanque) [inline General/Baselines.md]:
```suggestion
- `{optionalHostname}`: Optional server host, for private git servers
```

File: General/Baselines.md (hunk 1 above)
Comment (StevenCostiou) [reply General/Baselines.md]: I committed but since it is a server host url, why not "optionalHostURL" ?

File: General/Baselines.md (hunk 2 above)
Comment (StevenCostiou) [reply General/Baselines.md]: Since it is a server host url, why not "optionalHostURL" ?

File: General/Baselines.md (hunk 1 above)
Comment (juliendelplanque) [reply General/Baselines.md]: Because it is not a URL but only the hostname. See https://en.wikipedia.org/wiki/URL

Comment (StevenCostiou) [reply General/Baselines.md]: Ok then

**Pull request merged**